### PR TITLE
react/week1/Homework

### PR DESCRIPTION
**Pull request by Saidemm**

### CODE REVIEW

File: react/week1/todo-list/src/Todos.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <h1>Todo List</h1>
+            {props.todos.map((list) => {
+                return (
+                    <TodoDetails key={list.id} item={list}/>
+                )
+                }
+            )}
+        </ul>
+    )
+}
```
Comment (calm-prog) [inline react/week1/todo-list/src/Todos.jsx]: These 2 components could be stored in separate files. Then it's easier for others to get a sense of the structure of your app simply by looking at your components folder.

File: react/week1/todo-list/src/Todos.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function TodoDetails(props){
```
Comment (calm-prog) [inline react/week1/todo-list/src/Todos.jsx]: I think it's a better idea to call this something singular, eg. "Todo". Then it's clear that we are talking about a single unit regardless of how many props it receives. If it returned several components, then you could rightly put it in plural.

File: react/week1/todo-list/src/Todos.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const TodoList = (props) => {
+    return (
+        <ul>
+            <h1>Todo List</h1>
+            {props.todos.map((list) => {
+                return (
+                    <TodoDetails key={list.id} item={list}/>
+                )
+                }
+            )}
```
Comment (calm-prog) [inline react/week1/todo-list/src/Todos.jsx]: This is not super important, just a trick to shorten your code. If you don't use curly brackets then it is implied for the arrow function to return whatever you put after it. 

So you could write it this way on one line: 
`{props.todos.map(list) => <TodoDetails key={list.id} item={list}/>}`

If you need to write your code on several lines, then you need to wrap your JSX tag(s).
Here's an example: 
```
{props.todos.map(list) => (
   <div>
     <TodoDetails key={list.id} item={list}/>
   </div>
)}
```
If you are curious, learn more about the arrow function [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

**Review by calm-prog — COMMENTED** [review]
Well done! You kept it simple, but at the same time you demonstrated that you already understand the key concepts of React. That's a nice start! :)

There weren't any major issues in your code. However, I had some ideas how you could give a quick overview of the structure of your app by giving each component its own file and giving descriptive names to them. 

Let me explain this in the code below.

File: react/week1/todo-list/src/Todos.jsx (hunk 1 above)
Comment (Saidemm) [reply react/week1/todo-list/src/Todos.jsx]: Thanks for your time and very effective feedback.

File: react/week1/todo-list/src/Todos.jsx (hunk 2 above)
Comment (Saidemm) [reply react/week1/todo-list/src/Todos.jsx]: Done!

**Pull request merged**